packages/openid4vp/src/client-identifier-scheme/parse-client-identifier-scheme.ts

```diff
@@ -87,9 +93,23 @@ export function parseClientIdentifier(
         'x509_san_uri',
         'https',
         'pre-registered',
+        'web-origin',
       ] satisfies ClientIdScheme[]),
   }
 
+  if (isOpenid4vpAuthorizationRequestDcApi(request)) {
```

Comment:
Do we handle the case where client_id_Scheme is not web-origin? I think with a signed request that is possible.

Also, do we handle the case where the web-origin is used but response mode is not dc_api or dc_api.jwt (you can only use that client id with DC API i think?)